data_model/clusters/DoorLock.xml

```diff
@@ -0,0 +1,2296 @@
+        <optionalConform/>
+      </item>
+      <item value="6" name="DoorForcedOpen" summary="Forced Door Open under Door Locked Condition">
+        <optionalConform>
+          <feature name="DPS"/>
+        </optionalConform>
+      </item>
+      <item value="7" name="DoorAjar" summary="Door ajar">
+        <optionalConform>
+          <feature name="DPS"/>
+        </optionalConform>
+      </item>
+      <item value="8" name="ForcedUser" summary="Force User SOS alarm">
+        <optionalConform>
+          <feature name="USR"/>
+        </optionalConform>
+      </item>
+    </enum>
+    <enum name="CredentialRuleEnum">
+      <item value="0" name="Single" summary="Only one credential is required for lock operation">
+        <mandatoryConform>
+          <feature name="USR"/>
+        </mandatoryConform>
+      </item>
+      <item value="1" name="Dual" summary="Any two credentials are required for lock operation">
+        <optionalConform>
+          <feature name="USR"/>
+        </optionalConform>
+      </item>
+      <item value="2" name="Tri" summary="Any three credentials are required for lock operation">
+        <optionalConform>
+          <feature name="USR"/>
+        </optionalConform>
+      </item>
+    </enum>
+    <enum name="CredentialTypeEnum">
+      <item value="0" name="ProgrammingPIN" summary="Programming PIN code credential type">
+        <optionalConform/>
+      </item>
+      <item value="1" name="PIN" summary="PIN code credential type">
+        <mandatoryConform>
+          <feature name="PIN"/>
+        </mandatoryConform>
+      </item>
+      <item value="2" name="RFID" summary="RFID identifier credential type">
+        <mandatoryConform>
+          <feature name="RID"/>
+        </mandatoryConform>
+      </item>
+      <item value="3" name="Fingerprint" summary="Fingerprint identifier credential type">
+        <mandatoryConform>
+          <feature name="FGP"/>
+        </mandatoryConform>
+      </item>
+      <item value="4" name="FingerVein" summary="Finger vein identifier credential type">
+        <mandatoryConform>
+          <feature name="FGP"/>
+        </mandatoryConform>
+      </item>
+      <item value="5" name="Face" summary="Face identifier credential type">
+        <mandatoryConform>
+          <feature name="FACE"/>
+        </mandatoryConform>
+      </item>
+    </enum>
+    <enum name="DataOperationTypeEnum">
+      <item value="0" name="Add" summary="Data is being added or was added">
+        <mandatoryConform/>
+      </item>
+      <item value="1" name="Clear" summary="Data is being cleared or was cleared">
+        <mandatoryConform/>
+      </item>
+      <item value="2" name="Modify" summary="Data is being modified or was modified">
+        <mandatoryConform/>
+      </item>
+    </enum>
+    <enum name="DoorLockStatus">
+      <item value="2" summary="Entry would cause a duplicate credential/ID.">
```

Comment:
This is because they're in a weird table - they're status codes, which the SDK implements as enums. 

Tracking issue here: https://github.com/csa-data-model/projects/issues/335